services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/AutoRerouteSpec.groovy

```diff
@@ -89,27 +90,74 @@ class AutoRerouteSpec extends BaseSpecification {
         }
     }
 
-    def "Flow goes to 'Down' status when an intermediate switch is disconnected and there is no ability to reroute"() {
+    @Unroll
+    def "Flow goes to 'Down' status when src or dst switch is disconnected (#description)"() {
         requireProfiles("virtual")
 
-        given: "Two active not neighboring switches"
-        def switches = topology.getActiveSwitches()
-        def links = northboundService.getAllLinks()
-        def (Switch srcSwitch, Switch dstSwitch) = [switches, switches].combinations()
-                .findAll { src, dst -> src != dst }.find { Switch src, Switch dst ->
-            links.every { link ->
-                def switchIds = link.path*.switchId
-                !(switchIds.contains(src.dpId) && switchIds.contains(dst.dpId))
-            }
-        } ?: assumeTrue("No suiting switches found", false)
+        given: "A flow"
+        //TODO(ylobankov): Remove this code block once the issue #1464 is resolved.
+        if (description == "single-switch flow") {
+            assumeTrue("Test is skipped because of the issue #1464", false)
```

Comment:
assumeTrue("Test is skipped because of the issue #1464", description != "single-switch flow")